### ISSUE TITLE
TelInput should support `aria-*` props for accessibility

### ISSUE THREAD

**Issue by singingwolfboy** [gyurielf/svelte-tel-input#215]
I would like to use the [`aria-invalid`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid) and [`aria-errormessage`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage) props on the `TelInput` component, but it appears that _none_ of the ARIA props are supported. Can you support ARIA props on this component? It should be fine to just forward unrecognized props to the `<input>` tag.

**Comment by gyurielf** [comment]
Somehow I didn't receive notification about this issue.
I'll get back soon.
